### PR TITLE
fix: docs/standard/microservices-architecture/multi-container-microservice-net-applications/multi-container-applications-docker-compose.md

### PR DESCRIPTION
**Pull request by nschonni**
- interate -> iterate
- nedded -> needed
- ngend -> ngen'd

### CODE REVIEW

File: docs/standard/microservices-architecture/multi-container-microservice-net-applications/multi-container-applications-docker-compose.md [hunk 1]
```diff
@@ -440,30 +441,30 @@ A Dockerfile like this will work. However, you can substantially optimize your i
...
 
-The .NET team has been doing important work to make .NET Core and ASP.NET Core a container-optimized framework. Not only is .NET Core a lightweight framework with a small memory footprint; the team has focused on optimized Docker images for three main scenarios and published them in the Docker Hub registry at <span class="underline">microsoft/dotnet</span>, beginning with version 2.1:
+The .NET team has been doing important work to make .NET Core and ASP.NET Core a container-optimized framework. Not only is .NET Core a lightweight framework with a small memory footprint; the team has focused on optimized Docker images for three main scenarios and published them in the Docker Hub registry at *microsoft/dotnet*, beginning with version 2.1:
 
-1.  **Development**: Where the priority is the ability to quickly interate and debug changes and size is secondary.
+1. **Development**: Where the priority is the ability to quickly iterate and debug changes, and where size is secondary.
```
Comment (rpetrusha) [inline docs/standard/microservices-architecture/multi-container-microservice-net-applications/multi-container-applications-docker-compose.md]: I'm not sure that iterate is the right word here. Integrate seems to me to make more sense. @mvelosop, @CESARDELATORRE, is iterate correct?

File: docs/standard/microservices-architecture/multi-container-microservice-net-applications/multi-container-applications-docker-compose.md [hunk 2]
```diff
@@ -440,30 +441,30 @@ A Dockerfile like this will work. However, you can substantially optimize your i
...
 
-1.  **sdk**: for the development and build scenarios.
-2.  **runtime**: for the production scenario and
-3.  **runtime-deps**: for the production scenario of [self-contained applications](../../../core/deploying/index.md#self-contained-deployments-scd).
+1. **sdk**: for the development and build scenarios.
+2. **runtime**: for the production scenario and
+3. **runtime-deps**: for the production scenario of [self-contained applications](../../../core/deploying/index.md#self-contained-deployments-scd).
 
-Runtime images also provides automatic setting of aspnetcore\_urls to port 80 and the pre-ngend cache of assemblies; to help in getting faster startup.
+Runtime images also provides automatic setting of aspnetcore\_urls to port 80 and the pre-ngen'd cache of assemblies; to help in getting faster startup.
```
Comment (rpetrusha) [inline docs/standard/microservices-architecture/multi-container-microservice-net-applications/multi-container-applications-docker-compose.md]:
```suggestion
For faster startup, runtime images also automatically set aspnetcore\_urls to port 80 and use Ngen to create a native image cache of assemblies.
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
I've suggested alternate wording for one paragraph, @nschonni and @mvelosop, but I'm also not certain that iterate is the correct fix for interate.

File: docs/standard/microservices-architecture/multi-container-microservice-net-applications/multi-container-applications-docker-compose.md (hunk 1 above)
Comment (mvelosop) [reply docs/standard/microservices-architecture/multi-container-microservice-net-applications/multi-container-applications-docker-compose.md]: "iterate" is correct, this is an extract from the [Official .NET Docker images page](https://docs.microsoft.com/en-us/dotnet/standard/microservices-architecture/net-core-net-framework-containers/official-net-docker-images#during-development-and-build)

**Review by mvelosop — APPROVED** [review]
Thanks @rpetrusha, iterate is correct in this context.

LGTM @nschonni, you can squash-merge when you're ready.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**